homeassistant/components/airnow/__init__.py

```diff
@@ -15,14 +15,16 @@
 from homeassistant.helpers import device_registry as dr, entity_registry as er
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
 
-from .const import DOMAIN
+from .const import DOMAIN  # noqa: F401
```

Comment:
Why do you keep the import?

Reply:
It's used for the tests. I didn't want to change that to import from ’const’ instead as I usually prefer if all outside code only imports from the package root. That generally makes refactoring easier.

_Although that likely doesn't matter much in this case._